### DISCUSSION TITLE
Failing to get Readings

### DISCUSSION THREAD

**Discussion by Lynd** [NightscoutFoundation/xDrip#3100] [General]
Earlier today, I received a "sensor failed 7" error with my G7 sensor that persisted for hours.  Eventually I replaced the sensor and entered the new sensor code printed on the insertion device in the Transmitter ID field.  After 2 + hour warmup, I receive an error "not getting any [readings".]([url](url))   Further, there is a message indicating that my sensor ends today (the time of the old/replaced sensor).  

I'm sure I entered the correct G7 sensor ID printed insertion device.  

Note: In my stress trying to get this to work, I  may not have followed the correct xDrip steps to replace the sensor.

![1000006137](https://github.com/NightscoutFoundation/xDrip/assets/8272764/199a3cfb-a376-4445-9352-54b00607c52a)
![1000006138](https://github.com/NightscoutFoundation/xDrip/assets/8272764/858286e9-0e79-4696-be32-7fd4b12905c0)
![image000000 jpg](https://github.com/NightscoutFoundation/xDrip/assets/8272764/58be07f5-125e-4650-9a73-f698ac23e2a8)

**Comment by Navid200** [comment]
Does Dexcom app work?

**Comment by Lynd** [comment]
I just tested it.  No. It doesn't.

**Reply by Navid200** [reply]
If you have disabled collection on xDrip and Dexcom app does not work, there isn't anything we can do to resolve this.  
You can contact Dexcom.
